src/Apps/Artist/Routes/Overview/Components/WorksForSaleRail.tsx

```diff
@@ -39,6 +40,7 @@ const WorksForSaleRail: React.FC<WorksForSaleRailProps & {
         return (
           <FillwidthItem
             artwork={artwork.node}
+            contextModule={ContextModule.worksForSaleRail}
```

Comment:
```js
import { AnalyticsSchemaV2 } from "Artsy"
...
contextModule={AnalyticsSchemaV2.ContextModule.worksForSaleRail}
```
that way we have a consistent way to find and replace in the future aligned with the rest of our analytics imports.